internal/pkg/externalplugins/cherrypicker/cherrypicker_test.go

```diff
@@ -138,7 +186,34 @@ func (f *fghc) EnsureFork(forkingUser, org, repo string) (string, error) {
 	return repo, nil
 }
 
-var prFmt = `title=%q body=%q head=%s base=%s labels=%v assignees=%v`
+func (f *fghc) IsMember(org, user string) (bool, error) {
+	f.Lock()
+	defer f.Unlock()
+
+	switch user {
+	case patternErrWhenJudgeMember:
+		return false, errors.New("fake error")
+	default:
+		return sets.NewString(f.members...).Has(user) || f.isMember, nil
+	}
+}
+
+func (f *fghc) IsCollaborator(org, repo, user string) (bool, error) {
+	if strings.Contains(user, patternErrWhenIsCollaborator) {
```

Comment:
fghc instance is using in all test records in same test func, if a new plugin we should using gomock instead of fghc for free controlling and mocking.